src/core/buffers/representation/get_needed_segments.ts

```diff
@@ -253,13 +253,13 @@ function filterGarbageCollectedSegments(
 }
 
 /**
- * From the given SegmentInventory, filters the buffered Segment Object which
- * overlap with the given range.
+ * From the given SegmentInventory, filters the "playable" (decipherable, in
```

Comment:
Isn't it strange to tell that these segments are decipherable ?
The word implies that, if the content is cyphered, can be deciphered with the proper key by the browser.

Reply:
The proper formulation would be "not undecipherable" but it adds what appears to be a double negation!
But it's true that we do not check if they are "decipherable" so I will update that comment.

Reply:
Done